ansible/roles/operator-pipeline/templates/openshift/tasks/copy-image.yml

```diff
@@ -23,8 +27,15 @@ spec:
       script: |
         set -xe
 
+        # optionally tag destination image as latest
+        if [ "$(params.is_latest)" = "true" ]; then
```

Comment:
The if/else logic here is wrong. The "optional" means that the image is copied to the destination registry with a given tag and OPTIONALLY the latest tag is also created. This correct logic doesn't support both.

Reply:
logic is changed to reuse `skopeo copy` to retag it `latest` if condition true - original tag is kept in both cases